lib/src/git.rs

```diff
@@ -1383,88 +1383,29 @@ pub fn clone(
     Ok(stats)
 }
 
+#[tracing::instrument(skip(mut_repo, git_repo, callbacks))]
 pub fn fetch(
     mut_repo: &mut MutableRepo,
     git_repo: &git2::Repository,
-    remote_name: &str,
+    remote_names: &[&str],
```

Comment:
I feel it's a bit odd that `fetch()` takes multiple `remote_names`. (It doesn't make much sense to fetch the same `branch_names` from multiple remotes. It's kinda a limitation of the CLI.) So I think this can be moved to caller by using new `GitFetch` API.

Reply:
I didn't think of that. Thanks.

I've gone ahead to directly use the new `GitFetch` API in both the `clone` and `fetch` commands. The multiple remotes is handled in `git_fetch` in the cli.